sno/masters/assets/mc.yml

```diff
@@ -0,0 +1,67 @@
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  creationTimestamp: "2022-04-25T08:10:33Z"
```

Comment:
I'd say to clean up this file a little bit (removing for example the creationtimestamp, managedfields, etc.)